packages/kyc/test/unit/address.test.ts

```diff
@@ -0,0 +1,38 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+
+import { makeExecute } from '../../src/adapter'
+
+describe('address validations', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+
+  process.env.API_KEY = 'fake-api-key'
+
+  describe('validation error', () => {
+    const requests = [
+      {
+        name: 'endpoint is invalid',
```

Comment:
Please pay more attention, this test is not failing cause "endpoint is invalid", as you defined a default endpoint. It fails cause `method` is not being passed.

With the boilerplate it does not make much sense having unit tests for the inputs, better focus on `controllers` and `models` unit tests. Then make sure each input parameter is being tested on the integration ones.